app/src/main/java/org/kiwix/kiwixmobile/localFileTransfer/WifiDirectManager.kt

```diff
@@ -61,7 +60,8 @@ import javax.inject.Inject
 class WifiDirectManager @Inject constructor(
   private val activity: Activity,
   private val sharedPreferenceUtil: SharedPreferenceUtil,
-  private val alertDialogShower: AlertDialogShower
+  private val alertDialogShower: AlertDialogShower,
+  private val manager: WifiP2pManager? // Overall manager of Wifi p2p connections for the module
```

Comment:
we can just delete the comment "The WifiP2pManager manages wifi p2p" is a hella redundant sentence

Reply:
Okay okay, apologies for the offense